book/glossary.md

```diff
@@ -83,59 +101,86 @@ __HTML__.
 __Element__: Most __Nodes__ in the __DOM__ tree are Elements (except for
 text and the document object). (Inherits from __Node__.)
 
-__Focus__: the property of an __element__ being the highlight, or "focus",
+__Event__: A way for JavaScript to observe that something has happened on the
+document, and customize its results.
+
+__Focus__: Ahe property of an __element__ being the highlight, or "focus",
```

Comment:
```suggestion
__Focus__: The property of an __element__ being the highlight, or "focus",
```

I would not italicize `element` because some non-HTML elements like, say, the address bar can also have focus.